iso20022/modules/cash_management/camt.026.001.10.bal

```diff
@@ -58,8 +58,8 @@ public type MissingData1Choice record {|
 # + IncrrctInf - The incorrect information details
 public type MissingOrIncorrectData1 record {|
     AMLIndicator AMLReq?;
-    UnableToApplyMissing2[10] MssngInf?;
-    UnableToApplyIncorrect2[10] IncrrctInf?;
+    UnableToApplyMissing2[] MssngInf?;
```

Comment:
Is this change are done due to the filler values in Ballerina?

Reply:
Yes, there is constraints in the xml schema for the array length but the way it is done in these Bal records is incorrect because of the filler values